### DISCUSSION TITLE
S3 compatible

### DISCUSSION THREAD

**Discussion by caveman2003** [drakkan/sftpgo#1609] [General]
Hi all, 

just spend some time to get S3 compatible attached to SFTPgo... 

Used environment:
SFTPgo image: latest edge (Podman/Docker)
S3 compatible share provided by NetApp Storage Grid, self-signed SSL certificate.
After adding Bucket, Access Key, Access Secret, Endpoint, Check path-style and Check Skip TLS verify, uploading of files was not possible ... log show: Region missing ... well no Region is available on NetApp ...
After adding: Region = us-east-1 upload works....  

it would be useful if the latest docker image also have Check Skip TLS verify....

Thanks!!

**Comment by drakkan** [comment]
You just have to wait for v2.6.0 and if you can pay for a such storage you should also considering to [support us](https://github.com/drakkan/sftpgo?tab=readme-ov-file#sponsors) if you want to ensure SFTPGo stays healthy and maintained. Thank you
